.github/ISSUE_TEMPLATE.md

```diff
@@ -1,10 +1,12 @@
 _Before filing a new issue, please **provide the following information**._
 
+_If you think that your issue is an exploitable security vulnerability, please mail your bugreport to security@parity.io instead._
```

Comment:
maybe link `SECURITY.md`?

Reply:
Oh, great idea, will do (and will also revisit `SECURITY.md` to make sure it's up-to-date)